scripts/update_voice_data.py

```diff
@@ -0,0 +1,226 @@
+    "hr-HR": "GabrijelaNeural",
+    "hu-HU": "NoemiNeural",
+    "hy-AM": "AnahitNeural",
+    "id-ID": "GadisNeural",
+    "is-IS": "GudrunNeural",
+    "it-IT": "ElsaNeural",
+    "ja-JP": "NanamiNeural",
+    "jv-ID": "SitiNeural",
+    "ka-GE": "EkaNeural",
+    "kk-KZ": "AigulNeural",
+    "km-KH": "SreymomNeural",
+    "kn-IN": "SapnaNeural",
+    "ko-KR": "SunHiNeural",
+    "lo-LA": "KeomanyNeural",
+    "lt-LT": "OnaNeural",
+    "lv-LV": "EveritaNeural",
+    "mk-MK": "MarijaNeural",
+    "ml-IN": "SobhanaNeural",
+    "mn-MN": "BataaNeural",
+    "mr-IN": "AarohiNeural",
+    "ms-MY": "YasminNeural",
+    "mt-MT": "GraceNeural",
+    "my-MM": "NilarNeural",
+    "nb-NO": "IselinNeural",
+    "ne-NP": "HemkalaNeural",
+    "nl-BE": "DenaNeural",
+    "nl-NL": "ColetteNeural",
+    "pl-PL": "AgnieszkaNeural",
+    "ps-AF": "LatifaNeural",
+    "pt-BR": "FranciscaNeural",
+    "pt-PT": "RaquelNeural",
+    "ro-RO": "AlinaNeural",
+    "ru-RU": "SvetlanaNeural",
+    "si-LK": "ThiliniNeural",
+    "sk-SK": "ViktoriaNeural",
+    "sl-SI": "PetraNeural",
+    "so-SO": "UbaxNeural",
+    "sq-AL": "AnilaNeural",
+    "sr-RS": "SophieNeural",
+    "su-ID": "TutiNeural",
+    "sv-SE": "SofieNeural",
+    "sw-KE": "ZuriNeural",
+    "sw-TZ": "RehemaNeural",
+    "ta-IN": "PallaviNeural",
+    "ta-LK": "SaranyaNeural",
+    "ta-MY": "KaniNeural",
+    "ta-SG": "VenbaNeural",
+    "te-IN": "ShrutiNeural",
+    "th-TH": "AcharaNeural",
+    "tr-TR": "EmelNeural",
+    "uk-UA": "PolinaNeural",
+    "ur-IN": "GulNeural",
+    "ur-PK": "UzmaNeural",
+    "uz-UZ": "MadinaNeural",
+    "vi-VN": "HoaiMyNeural",
+    "wuu-CN": "XiaotongNeural",
+    "yue-CN": "XiaoMinNeural",
+    "zh-CN": "XiaoxiaoNeural",
+    "zh-CN-henan": "YundengNeural",
+    "zh-CN-liaoning": "XiaobeiNeural",
+    "zh-CN-shaanxi": "XiaoniNeural",
+    "zh-CN-shandong": "YunxiangNeural",
+    "zh-CN-sichuan": "YunxiNeural",
+    "zh-HK": "HiuMaanNeural",
+    "zh-TW": "HsiaoChenNeural",
+    "zu-ZA": "ThandoNeural",
+}
+
+
+async def main():
+    async with aiohttp.ClientSession() as session:
+        await run(session)
+
+
+async def run(session):
+    token = None
+
+    while not token:
+        print("Please enter your Azure token:")
+        token = input().strip()
+
+    voices = await session.get(
+        LIST_VOICES_URL, headers={"Authorization": f"Bearer {token}"}
+    )
+    if voices.status != 200:
+        print(f"Failed to fetch voices: {voices.status}")
+        return
+    voices_data = await voices.json()
+
+    data = {}
+    for voice in voices_data:
+        if voice["Status"] != "GA":
+            continue
+        locale = voice["Locale"]
+        voice_name = voice["ShortName"][len(locale) + 1 :]
+        voice_info = {}
+
+        # If we have no default for the voice, make the first one the default
+        DEFAULTS.setdefault(locale, voice_name)
+
+        if DEFAULTS[locale] == voice_name:
+            voice_info["default"] = True
```

Comment:
Why should the lib dictate the default over the consumer?

Reply:
Because it does today, and it will keep doing that. This data is meant to replace `TTS_VOICES` and so has to contain the same data.

Reply:
<https://github.com/home-assistant/core/blob/dev/homeassistant/components/cloud/tts.py#L41> but the defaults are defined here (in the consumer)?

Reply:
I'll be damned, then why do we still have that comment 😄  I will drop it.

Reply:
It was added to core recently, so generating the values like this PR aims to do would be viable.
It was just not properly cleaned up afterwards.